config/routes.rb

```diff
@@ -0,0 +1,23 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  get '/shopping_list', to: 'foods#shopping_list', as: 'shopping_list'
+
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+    get '/public_recipes', to: 'recipes#public_recipes'
+    delete '/recipes/:id/:food_id', to: 'recipes#remove_food_from_recipe', as: 'recipe_food'
+
+
+    resources :recipes, only: %i[index show update destroy] do
+      resources :recipe_foods, only: [:index, :show, :new, :create]
+    end
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work so far but kindly double-check the requirements because you should have all the crud operations except the update I am not able to create a recipe in your app so because of that I cannot see the recipe or anything :fearful: Kindly make a way so a user can create a recipe in your app :+1: